src/java/org/apache/cassandra/tools/NodeProbe.java

```diff
@@ -1544,9 +1544,15 @@ public void loadNewSSTables(String ksName, String cfName)
         ssProxy.loadNewSSTables(ksName, cfName);
     }
 
-    public List<String> importNewSSTables(String ksName, String cfName, Set<String> srcPaths, boolean resetLevel, boolean clearRepaired, boolean verifySSTables, boolean verifyTokens, boolean invalidateCaches, boolean extendedVerify, boolean copyData)
-    {
-        return getCfsProxy(ksName, cfName).importNewSSTables(srcPaths, resetLevel, clearRepaired, verifySSTables, verifyTokens, invalidateCaches, extendedVerify, copyData);
+    public List<String> importNewSSTables(String ksName, String cfName, Set<String> srcPaths, boolean resetLevel,
```

Comment:
I created new method instead of modifying the old one to be backward compatible / not break stuff for people using the older version already.

Reply:
So your comment says you added a new one, but it looks like you modified the old one... I don't know if there's anything other than the `Import` command that actually uses it, but wanted to check if it was intentional that this comment doesn't match the code - or did you mean you made a new one in `ColumnFamilyStore/ColumnFamilyStoreMBean` which would also make sense, and which you did... so maybe I just misunderstood this PR comment?

Reply:
@JeetKunDoug correct, I created new one on ColumnFamilyStore/MBean but I changed the one on NodeProbe. That is just fine, because NodeProbe one is "internal" to Cassandra. Nobody depends on _NodeProbe_ across versions ... Does that make sense?